### ISSUE TITLE
Replication ACKS doesn't break connection to remote peer

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#3025]
1. Setup master <-> master replication for two replicas
2. Power off replica1 
3. Replica2 will show 'follow' status for eternity until connection is break by kernel tcp stack

100% reproducibility using two qemu-kvm boxes (virt-manager).

Workaround 1:

```
tarantool> replication = box.cfg.replication
tarantool> box.cfg  { replication = "" }
tarantool> box.cfg. { replication = replication }
```

Workaround 2:

```
sysctl -w net.ipv4.tcp_keepalive_time = 60
sysctl -w net.ipv4.tcp_keepalive_intvl = 5
sysctl -w net.ipv4.tcp_keepalive_probes = 6
```

**Comment by locker** [comment]
Fixed by 99c7a971df04d7234ee88da501b291497c4dd43f

**Comment by rtsisyk** [comment]
This fix was needed in 1.7.x.

**Comment by kostja** [comment]
There is no 1.7.x, Roman, 1.7.7 will become 1.9.0
